src/br/com/ciandt/csartini/service/ComentarioService.java

```diff
@@ -0,0 +1,57 @@
+package br.com.ciandt.csartini.service;
+
+/**
+* Método que realiza a remoção do comentário
+* Exemplo: {@code Boolean seraExcluidoSe = comentarioRemovido(comentario)
+* @param comentario {@link} variavel
+* @return se verdadeiro ou falso
+* 
+* @author csartini
+**/
+
+/**
+ * metodo construtor acima
+ * 
+ * @author csartini
+ *
+ */
+public class ComentarioService {
```

Comment:
Rever o comentário acima. Neste momento, devemos inserir um comentário da classe.
```suggestion
/**
* Classe de serviço para comentários
* @author csartini
**/
public class ComentarioService {
```